workflows-scripts/inventory/item-load-inventory/README.md

```diff
@@ -0,0 +1,40 @@
+## Workflow steps:
+### Pre-requisite:
+1. Open Inventory App
+2. Select Instance tab in top left
+3. Select Keyword(title, contributor, identifier) in the drop down and type "aba" in search field
+
+
+## Workflow steps:
+##### Workflow for inventory_loadItems.jmx
+1. Select the third instance "ABA 7th National Conference on Law Office Economics 
```

Comment:
These are the steps describing the workflow that a user takes, could you please explain in English to how the JMeter script works by using the instanceIDs, holdingsIDs, items barcodes, etc, since this is a fairly complex workflow? How was the script designed, how does the JMeter script work? I think this is a useful information to include.